### ISSUE TITLE
benchmark: GRU+SVM for MNIST dataset

### ISSUE THREAD

**Issue by AFAgarap** [zalandoresearch/fashion-mnist#30]
I saw you updated the `README` to report on comparison between the accuracy for your dataset and MNIST, so I did a benchmark of my proposed GRU+SVM model (issue #8) on MNIST as well. 

Here are the results:
```
Epoch : 0 completed out of 10, loss : 35.062599182128906, accuracy : 0.9296875
Epoch : 1 completed out of 10, loss : 20.101139068603516, accuracy : 0.9609375
Epoch : 2 completed out of 10, loss : 11.310111999511719, accuracy : 0.984375
Epoch : 3 completed out of 10, loss : 14.316896438598633, accuracy : 0.96875
Epoch : 4 completed out of 10, loss : 13.816293716430664, accuracy : 0.9609375
Epoch : 5 completed out of 10, loss : 8.049131393432617, accuracy : 0.984375
Epoch : 6 completed out of 10, loss : 10.147947311401367, accuracy : 0.984375
Epoch : 7 completed out of 10, loss : 8.27488899230957, accuracy : 0.9921875
Epoch : 8 completed out of 10, loss : 21.153032302856445, accuracy : 0.9609375
Epoch : 9 completed out of 10, loss : 10.881654739379883, accuracy : 0.9765625
Accuracy : 0.9658001661300659
```

The source may be found here, in [my GitHub Gist](https://gist.githubusercontent.com/AFAgarap/92c1c4a5dd771999b0201ec0e7edfee0/raw/547c3d821965f4fbeb6c13c2bbed291bf63ec8ef/gru_svm_mnist.py). The hyper-parameters used were as follows:
```
BATCH_SIZE = 128
CELL_SIZE = 32
HM_EPOCHS = 10
LEARNING_RATE = 0.01
NUM_CLASSES = 10
SVM_C = 0.5
```

Trained using `tf.train.AdamOptimizer()`, and used the `tf.nn.static_rnn()`.

**Comment by AFAgarap** [comment]
Using the same GRU+SVM model, but now, with Dropout, here's the result on MNIST dataset:

```
Epoch : 0 completed out of 10, loss : 183.0938720703125, accuracy : 0.87109375
Epoch : 1 completed out of 10, loss : 83.48553466796875, accuracy : 0.953125
Epoch : 2 completed out of 10, loss : 96.95372009277344, accuracy : 0.94921875
Epoch : 3 completed out of 10, loss : 36.80517578125, accuracy : 0.984375
Epoch : 4 completed out of 10, loss : 57.431053161621094, accuracy : 0.97265625
Epoch : 5 completed out of 10, loss : 31.22872543334961, accuracy : 0.98046875
Epoch : 6 completed out of 10, loss : 33.04032516479492, accuracy : 0.98046875
Epoch : 7 completed out of 10, loss : 31.262266159057617, accuracy : 0.984375
Epoch : 8 completed out of 10, loss : 20.89887046813965, accuracy : 0.98828125
Epoch : 9 completed out of 10, loss : 26.805370330810547, accuracy : 0.98828125
Accuracy : 0.9761001467704773
```

The following were the hyper-parameters used:

```
BATCH_SIZE = 256
CELL_SIZE = 256
DROPOUT_P_KEEP = 0.85
EPOCHS = 10
LEARNING_RATE = 1e-3
SVM_C = 1
```

Trained using `tf.train.AdamOptimizer()`, and used `tf.nn.dynamic_rnn()`. The source may be found [here](https://gist.githubusercontent.com/AFAgarap/92c1c4a5dd771999b0201ec0e7edfee0/raw/af3ffa1a402a6860fe2a1eece6203fbb578bf24b/gru_svm_mnist_dropout.py).
